lib/tasks/migrate.rake

```diff
@@ -117,6 +117,11 @@ namespace :migrate do
         .update_all(lifecycled_at: Time.current.to_s(:db))
       end
     end
+
+    desc "Fill live_project"
+    task fill_live_project: :environment do
+      Classification.where(live_project: nil).update_all("live_project = (metadata->>'live_project')::boolean")
```

Comment:
This one takes a bunch of minutes, and I'm not sure if it does a table lock (at least for writes). Any thoughts on how to do it better?